digitalocean/database/resource_database_postgres_config.go

```diff
@@ -0,0 +1,626 @@
+package database
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/digitalocean/godo"
+	"github.com/digitalocean/terraform-provider-digitalocean/digitalocean/config"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func ResourceDigitalOceanDatabasePostgreSQLConfig() *schema.Resource {
+	return &schema.Resource{
```

Comment:
```suggestion
	return &schema.Resource{
		CreateContext: resourceDigitalOceanDatabasePostgreSQLConfigCreate,
		ReadContext:   resourceDigitalOceanDatabasePostgreSQLConfigRead,
		UpdateContext: resourceDigitalOceanDatabasePostgreSQLConfigUpdate,
		DeleteContext: resourceDigitalOceanDatabasePostgreSQLConfigDelete,
		Importer: &schema.ResourceImporter{
			State: resourceDigitalOceanDatabasePostgreSQLConfigImport,
		},
```

Need to reference the CRUD functions here